PyRDF/CallableGenerator.py

```diff
@@ -49,20 +88,21 @@ def mapper(node_cpp, node_py=None):
                 The current state's ROOT CPP node. Initially
                 this should be fed a value.
 
+            node_py (optional)
+                The current state's PyRDF node. If `None`,
+                it takes the value of `self.head_node`.
+
             Returns
             -------
-                tuple
-                    This tuple consists of 2 lists, the first one
-                    consisting of output values of action nodes and
-                    the second one of action nodes in the same order
-                    as that of the values list.
+                list
+                    This is a list of values in the order that they
```

Comment:
And here I would say: "A list of `TResultPtr` objects in DFS order of their corresponding actions in the graph."

@shravan97 , note that here we do not execute anything, the whole event loop will be triggered when accessing one of the `TResultPtr`s later in the backend.

Reply:
Yup @etejedor . Will change this 👍